src/tlog/verify/body.ts

```diff
@@ -26,7 +26,9 @@ export function verifyTLogBody(
   bundleContent: sigstore.Bundle['content']
 ): boolean {
   const { kind, version } = entry.kindVersion;
-  const body: EntryKind = JSON.parse(entry.canonicalizedBody.toString('utf8'));
+  const body = JSON.parse(
+    entry.canonicalizedBody.toString('utf8')
+  ) as EntryKind;
```

Comment:
Yeah seems like both work because `JSON.parse` returns `any`, but with the reset it changed it to `unknown` so had to explicitly cast it. Changed it back to how it was now that we're not including the reset lib 👍